scripts/auth.js

```diff
@@ -0,0 +1,23 @@
+import btoa from "btoa";
+
+
+export default class Auth {
+  constructor(events) {
+    this.events = events;
+    this.server = '';
+  }
+
+  configure(config) {
+    this.server = config.server;
+  }
+
+  authenticate() {
+    // Take username from location hash (With default value):
+    const user = window.location.hash = (window.location.hash.slice(1) || "public-demo");
```

Comment:
`window` as an internal dependency makes this very hard to unit test if we want to; how about supporting injecting it through a constructor option?